src/errorline.jl

```diff
@@ -0,0 +1,179 @@
+@userplot ErrorLine
+""" 
+# StatsPlots.errorline(x, y, arg): 
+    Function for parsing inputs to easily make a ribbons (https://ggplot2.tidyverse.org/reference/geom_ribbon.html) 
+    or errorbar (https://www.mathworks.com/help/matlab/ref/errorbar.html) plot while allowing for easily controlling error
+    type and NaN handling.
+
+# Inputs: default values are indicated with *s
+
+    x (vector, unit range) - the values along the x-axis for each y-point
+
+    y (matrix [x, repeat, group]) - values along y-axis wrt x. The first dimension must be of equal length to that of x.
+        The second dimension is treated as the repeated observations and error is computed along this dimension. If the 
+        matrix has a 3rd dimension this is treated as a new group.
+
+    ErrorStyle (symbol - *:Ribbon* or :Classic) - determines whether to use a ribbon style or stick style error representation.
```

Comment:
```suggestion
    error_style (symbol - *:ribbon* or :classic) - determines whether to use a ribbon style or stick style error representation.
```
In julia usually only types and modules have upper CamelCase

Reply:
Same applies for the other keyword arguments